frontend/app/templates/components/data-component/csv-component.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="field">
     <label>Upload CSV</label>
-    <input name="csv" onchange={{action "encodeCSV"}} type="file">
+    <input name="csv" onchange={{action "encodeCSV"}} type="file" accept=".csv">
```

Comment:
PR related to this change already Merged.

Reply:
@ParthS007  It will overwrite that . Will it make any  issues??

Reply:
I don't know why this is here